dotnet/samples/Concepts/TextSearch/AzureAISearchExample.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This sample is in `TextSearch` folder, while we also have just `Search` folder, which contains some text search logic inside :) It would be nice to align these samples to avoid confusion.

Reply:
I did that deliberately in this PR to keep the new samples seperate from the existing code.